c/pthread-memsafety/list1_false-valid-deref.c

```diff
@@ -43,7 +43,8 @@ int main(int argc, char **argv) {
   pthread_t id1, id2;
   struct item *list = ((void *)0);
   pthread_create(&id1, NULL, build, list);
+  pthread_join(id1, NULL);
   pthread_create(&id2, NULL, delete, list);
-
+  pthread_join(id1, NULL);
```

Comment:
This is inconsistent with the preprocessed version - should be `id2`

Reply:
Ops! Fixed.